scripts/truffle_project.py

```diff
@@ -28,28 +28,33 @@
 
 
 class TruffleProject(project.Project):
-    """A project that uses the truffle development environment to compile the project."""
+    """A project that uses the truffle development environment to compile the
+    project."""
 
-    def __init__(self, project_root, pretty_output):
-        super().__init__(project_root, pretty_output)
+    def __init__(self, project_root, clargs):
+        super().__init__(project_root, clargs)
         self.build_dir = self.project_root / pathlib.Path("build/contracts/")
 
     def compile_(self, compilation_output):
         with utils.working_directory(self.project_root):
             try:
-                subprocess.check_output(["truffle", "compile"],
-                                        stderr=subprocess.STDOUT)
-            except subprocess.CalledProcessError as e:
-                logging.error("Error compiling truffle project.")
+                output = subprocess.check_output(["truffle", "compile"],
+                                                 universal_newlines=True,
+                                                 stderr=subprocess.PIPE)
```

Comment:
Given that the documentation ( https://docs.python.org/3/library/subprocess.html?highlight=subprocess#subprocess.check_output ) says to write it `stderr=subprocess.STDOUT`, write it like this here as well (it's the same in this case).

Reply:
Sure